parser.go

```diff
@@ -549,9 +549,7 @@ func (p *parser) parseInclude(t *token) (bool, Result, error) {
 	  |                                 |                                 |
 	  | none                            | return permerror                |
 	  +---------------------------------+---------------------------------+
-	*/
-
-	if err != nil {
+	*/if err != nil {
```

Comment:
this looks weird.

Reply:
It does, and it does not look like a feature of the `gofumpt`